spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/verify_credentials_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager do
+  it ".ems_type" do
+    expect(described_class.ems_type).to eq("ibm_cloud_power_virtual_servers")
+  end
+
+  let(:ems) do
+    uid_ems  = Rails.application.secrets.ibm_cloud_power[:cloud_instance_id]
+    auth_key = Rails.application.secrets.ibm_cloud_power[:api_key]
```

Comment:
@jaywcarman We just merged the VcrSecrets code, so this will have to follow the new pattern.

```suggestion
    uid_ems  = VcrSecrets.ibm_cloud_power.cloud_instance_id
    auth_key = VcrSecrets.ibm_cloud_power.api_key
```